mscore/timeline.cpp

```diff
@@ -73,6 +73,7 @@ TDockWidget::TDockWidget(QWidget* w)
       setMinimumHeight(50);
 
       setWindowTitle(tr("Timeline"));
+      setObjectName("timeline");
```

Comment:
It's not entirely homogenous... but most of the object name starts with a capital . Could you change it to "Timeline" please?